torch/export_utils.py

```diff
@@ -0,0 +1,18 @@
+from torch._export.constraints import constrain_as_size, constrain_as_value, dynamic_dim
+from torch._export.exported_program import ExportedProgram, ExportGraphSignature
+from torch._export.pass_base import ExportPassBase
+from torch._export.serde.serialize import deserialize, serialize
+from torch._export.utils import register_dataclass_as_pytree_node
+
+
+__all__ = [
+    "dynamic_dim",
+    "ExportedProgram",
+    "ExportGraphSignature",
+    "serialize",
```

Comment:
What exactly is the output format of serialize? Is it BC? (I see it's in a folder named serde; is there any relationship to Rust serde?

Reply:
We have a high level serialization design doc here: https://docs.google.com/document/d/1N8-fY3QAnxHTBhNkM9VdfDdQk-NJ5wJ5QVOVLyWiGYw/edit

It has basic capability to address operator level BC issues, but we haven't put too much thought into the overall schema level BC. We may end up with a version number design.

The name is not related to rust. It's just abbreviation for ser/deserialization